core/io/translation_loader_po.cpp

```diff
@@ -110,6 +116,9 @@ RES TranslationLoaderPO::load_translation(FileAccess *f, Error *r_error, const S
 		}
 
 		if (l == "" || l.begins_with("#")) {
+			if (l.ends_with("fuzzy")) {
```

Comment:
There are many kinds of attributes that can go there, not only `fuzzy`, so it's not guaranteed to be the end of the string: https://www.gnu.org/software/gettext/manual/html_node/PO-Files.html
So I don't think you'll get around a `l.find("fuzzy") != -1` here :)